src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -283,3 +283,16 @@ interface Float32Array {
 interface Float64Array {
     readonly [Symbol.toStringTag]: "Float64Array";
 }
+
+interface ArrayConstructor {
+    readonly [Symbol.species]: ArrayConstructor;
+}
+interface MapConstructor {
+    readonly [Symbol.species]: MapConstructor;
+}
+interface SetConstructor {
+    readonly [Symbol.species]: SetConstructor;

```

Comment:
[Per the spec](https://tc39.github.io/ecma262/#sec-get-set-@@species), the property type should be `this`.